bin/indexer

```diff
@@ -4,24 +4,30 @@ const fs = require('fs');
 const execa = require('execa');
 const path = require('path');
 const os = require('os');
+const reindex_delay = process.env.REINDEX_DELAY || 300000;
```

Comment:
I suppose we could even set this to something lower than 5 mins now (eg 1-2 mins?) to reduce release latency (xref https://github.com/heroku/languages-github-actions/pull/28#discussion_r1238940026)

Reply:
Yeah, I'll go adjust this.

Reply:
https://github.com/buildpacks/registry-api/pull/115/commits/76b95c24c0995ef051b445b1dbcf35b96423e882